docs/src/dev/understanding_traces.md

```diff
@@ -0,0 +1,39 @@
+# Understanding Traces
+
+yk can print the traces it has created to `stdout` to help with debugging.
```

Comment:
Is it `stdout` or `stderr`? I though the latter.

Link to `YKD_PRINT_IR` docs?

Reply:
IMHO it should be stderr, but it's actually stdout.

Reply:
I beg to differ :)

https://github.com/ykjit/yk/blob/9b66acabf9559fa2a905d15065d1c9eaecf3523a/ykllvmwrap/src/ykllvmwrap.cc#L187

Reply:
I was scratching my head -- then I realised that I've been mostly piping stuff via ssh, and apparently that's putting stderr to stdout on my local machine? I didn't expect that!

Fixed in 49bfbba.

Reply:
Yeah, not what I expected! Fixed the code example in b7b4d60.